docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md

```diff
@@ -88,15 +88,16 @@ Select this authentication option if the vendor API requires bearer authenticati
 
 #### OAuth 2.0 Client Credentials
 
-Select this authentication option if the vendor API allows [OAuth 2 Client Credentials Grant Type](https://oauth.net/2/grant-types/client-credentials/) as a form of authentication. Please be aware, this is specifically for APIs that support the `Client Credentials` grant type and **NOT** any other form of OAuth grant type. 
+Select this authentication option if the vendor API allows [OAuth 2 Client Credentials Grant Type](https://oauth.net/2/grant-types/client-credentials/) as a form of authentication. Please be aware, this is specifically for APIs that support the `Client Credentials` grant type and **NOT** any other form of OAuth grant type.
```

Comment:
We restrict the use of [bold text to UI elements](https://help.sumologic.com/docs/contributing/style-guide/#bold).

```suggestion
Select this authentication option if the vendor API allows [OAuth 2 Client Credentials Grant Type](https://oauth.net/2/grant-types/client-credentials/) as a form of authentication. Please be aware, this is specifically for APIs that support the `Client Credentials` grant type and NOT any other form of OAuth grant type.
```